File: js/find.js
const { heroes } = require('./data.js');

//The .find() method will return the frst element in the array that matches a test you provide.
function isHulk(hero) {
    return hero.name === 'Hulk';
   }
var hulk = heroes.find(isHulk);

console.log(hulk);
//node find.js 
//expected output
//{ name: 'Hulk', strength: 90000 }

File: py/find.py
import data

def is_hulk(hero):
    return hero['name'] == 'Hulk'

hulk = next((hero for hero in data.heroes if is_hulk(hero)), None)

print(hulk)
#$ python3 find.py
#expected output
#{ name: 'Hulk', strength: 90000 }